lib/pengine/tests/status/set_working_set_defaults_test.c

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2022 the Pacemaker project contributors
+ *
+ * The version control history for this file may have further details.
+ *
+ * This source code is licensed under the GNU Lesser General Public License
+ * version 2.1 or later (LGPLv2.1+) WITHOUT ANY WARRANTY.
+ */
+
+#include <crm_internal.h>
+#include "mock_private.h"
+
+#include <stdarg.h>
+#include <stddef.h>
+#include <stdint.h>
+#include <setjmp.h>
+#include <cmocka.h>
+
+#include <crm/pengine/internal.h>
+#include <crm/pengine/pe_types.h>
+#include <crm/pengine/status.h>
+
+static void
+check_defaults(void **state) {
+    pe_working_set_t *data_set = calloc(1, sizeof(pe_working_set_t));
+
+    set_working_set_defaults(data_set);
+
+    assert_null(data_set->priv);
+    assert_int_equal(data_set->order_id, 1);
+    assert_int_equal(data_set->action_id, 1);
+    assert_int_equal(data_set->no_quorum_policy, no_quorum_stop);
+    assert_int_equal(data_set->flags,
+                     pe_flag_stop_rsc_orphans|pe_flag_symmetric_cluster|pe_flag_stop_action_orphans);
```

Comment:
You can either check that at least these or set, or have two checks depending on the value of PCMK__CONCURRENT_FENCING_DEFAULT (it's set via a configure option), since another flag will be set if that is true

Reply:
Fixed.